### PR TITLE
+ qfs 0.5

### PR DESCRIPTION
**Pull request by ygrek**

### CODE REVIEW

**Comment by ygrek** [comment]
those builds that failed - did so because of travis net problems

**Comment by avsm** [comment]
Looks like this failed on 4.01.0 since it needs `lwt.ppx` : add a compiler constraint?

```
# ocamlfind: Package `lwt.ppx' not found
```

File: packages/qfs/qfs.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "base-bytes"
+  "base-unix"
+  ("extlib" | "extlib-compat")
+  "lwt"
+  "ocamlfind" {build & >= "1.5"}
+]
+available: [ ocaml-version >= "4.01.0" ]
```
Comment (avsm) [inline packages/qfs/qfs.0.5/opam]: this needs to be `>=4.02.0` for lwt.ppx

**Comment by ygrek** [comment]
drat, missed the correct ocaml version (by memory) %)

**Comment by avsm** [comment]
thanks!

**Pull request merged**